.github/workflows/deploy-enterprise-test.yml

```diff
@@ -31,7 +31,6 @@ jobs:
               job_spec/INSAR_ISCE_BURST.yml
               job_spec/WATER_MAP.yml
               job_spec/WATER_MAP_EQ.yml
-              job_spec/S1_CORRECTION_ITS_LIVE.yml
```

Comment:
this job isn't available outside of the ITS_LIVE deployments, so we don't need to test it here.